javascript.js

```diff
@@ -51,10 +51,69 @@ getFunctionAdd.addEventListener("click", function(){
 
 function removeElement(parameter) {
   if(parameter != "No") {
-    console.log("getNumberArticle", parameter);
     let transform = document.getElementsByClassName(parameter)[0];
     transform.remove();
   }
 }
 
 removeElement("No");
+
+var getDataLocal = JSON.parse(localStorage.getItem('List'));
+
+if(getDataLocal != null) {
+  var validation = getDataLocal;
+}
+
+if(getDataLocal !== null && validation.length <= 2) {
+  getDataLocal;
+} else if(getDataLocal === null) {
+  getDataLocal;
+}
+
+function localStorageFct(parameter) {
+  if(parameter === "No") {
+
+    let getDataLocalStorage = JSON.parse(localStorage.getItem('List'));
+    let parseObjt = getDataLocalStorage;
+    console.log(parseObjt);
+    let getListAllBooks = document.getElementById('list_books');
```

Comment:
- [ ] Please don't forget to remove unnecessary lines of code like `console.log` so that your code is cleaner and more professional.

Reply:
Unnecessary code removed.